pkg/apis/options/load.go

```diff
@@ -140,10 +141,6 @@ func isUnexported(name string) bool {
 
 // LoadYAML will load a YAML based configuration file into the options interface provided.
 func LoadYAML(configFileName string, into interface{}) error {
-	v := viper.New()
-	v.SetConfigFile(configFileName)
-	v.SetConfigType("yaml")
-	v.SetTypeByDefaultValue(true)
```

Comment:
I'm surprised we are getting rid of these? That said, were they actually being used?

Reply:
the viper object is not used again after these 4 methods, and it is not returned either.